docs/UserGuide.md

```diff
@@ -230,74 +230,107 @@ There is no need for you to save manually.
-* When editing tags, the existing tags of the student will be removed i.e adding of tags is not cumulative.
-* You can remove all the student’s tags by typing `tag/` without
+* When editing tags, the existing tags of the student will be removed and replaced by the newly typed in tags.
+* You can remove all the tags of a student by typing `tag\` without
     specifying any tags after it.
 
-Examples:
-*  `edit-student 1 t/johndoe e/johndoe@example.com` Edits the telegram username and email address of the 1st student to be `johndoe` and `johndoe@example.com` respectively.
-*  `edit-student 2 n/Betsy Crower tag/` Edits the name of the 2nd student to be `Betsy Crower` and clears all existing tags.
+Other examples:
+*  `edit-student 1 t\smartlex e\yeohalex@example.com` \
+Edits the telegram username and email address of the 1st student to `smartlex` and `yeohalex@example.com` respectively.
+*  `edit-student 2 n/Betsy Yu tag/` \
+Edits the name of the 2nd student to `Betsy Yu` and clears all existing tags.
 
 #### Finding student by name : `find-student`
 
-Finds students whose names contain any of the given keywords.
+If you would like to search for a student by name, use this command.
 
 Format: `find-student KEYWORD [MORE_KEYWORDS]`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Students matching at least one keyword will be returned (i.e. **OR** search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+For example:
+* `Roy` requested for supplementary notes in class today.
+You want to email him these notes.
+Type `find-student roy` and press <kbd>Enter</kbd>.
 
-Examples:
-* `find-student John` returns `john` and `John Doe`
-* `find-student alex david` returns `Alex Yeoh`, `David Li`<br>
+  ![FindStudentBefore](images/ugimages/FindStudentBefore.png)
+
+  ![FindStudentAfter](images/ugimages/FindStudentAfter.png)
+
+Constraints:
+* The search is case-insensitive. For instance, `roy` will match `Roy`.
+* The order of the keywords does not matter. e.g. `Yeoh Alex` will match `Alex Yeoh`.
+* Only the name is searched.
+* Only full words will be matched. For instance, `Irfa` will not match `Irfan`.
+* Students matching at least one keyword will be returned. For instance, `Alex Li` will return `Alex Yeoh` and `David Li`
 
 #### Deleting a student : `delete-student`
 
-Deletes the specified student from the application.
+If you would like to delete a student, you can make use of this command.
 
 Format: `delete-student INDEX`
-* Deletes the student at the specified `INDEX`.
+
+For example:
+* One of your students, `Bernice`, has swapped `CS203T Tutorial` slots and you no longer teach her.
+Hence, you want to delete her entry from Tutor's Pet. You type in `delete-student 2` and press <kbd>Enter</kbd>.
```

Comment:
Perhaps it will be good to also mention that Bernice is the second student on the list?